src/scss/components/postLanding/info/_info.scss

```diff
@@ -6,19 +6,19 @@
 
     &__stat-container {
         position: relative;
-        padding-left: 25em;
+        padding-left: calc(24em + 4vw);
```

Comment:
don't use padding to center things like this; big no-no and it'll never be responsive. here's easier ways to center: https://css-tricks.com/centering-css-complete-guide/